tests/TestString.cfc

```diff
@@ -4,10 +4,10 @@ component extends = "mxunit.framework.TestCase" {
 		variables.string = new util.String();
 	}
 
-	function testEscapeJSONcharacters() {
+	function testEncodeForJSON() {
```

Comment:
i've been doing test_functionName to isolate the method name from the "test" prefix - i'm not sure what best practice is, but you'll see that treatment elsewhere. additionally - all the OSS code is assumed to live within a "lib" package - so your unit test should reflect that (on line 4 - lib.util.String) - if someone runs the full util suite, they'll have a bad time.

Reply:
I second the test_functionName convention